Rubberduck.CodeAnalysis/Inspections/Concrete/ProcedureNotUsedInspection.cs

```diff
@@ -20,25 +20,37 @@ namespace Rubberduck.CodeAnalysis.Inspections.Concrete
     /// <remarks>
     /// Not all unused procedures can/should be removed: ignore any inspection results for 
     /// event handler procedures and interface members that Rubberduck isn't recognizing as such.
+    /// Public procedures of Standard Modules are not flagged by this inspection regardless of
+    /// the presence or absence of user code references.
     /// </remarks>
     /// <example hasResult="true">
     /// <module name="MyModule" type="Standard Module">
     /// <![CDATA[
     /// Option Explicit
     /// 
+    /// 'No inspection result
     /// Public Sub DoSomething()
     ///     ' macro is attached to a worksheet Shape.
     /// End Sub
+    /// 
+    /// 'Flagged by inspection
```

Comment:
Let's keep the example simples, and add more of them if needed - the procedure that doesn't trigger the inspection should be in an example that doesn't trigger the inspection. The website will list all modules under a collapsible heading for each example:

![image](https://user-images.githubusercontent.com/5751684/81630077-b0fce380-93d2-11ea-97c9-4867c34d5f49.png)

Reply:
Simplified the current examples and added one for Class Module procedure(s) scenario.